src/ui/public/getting_started/registry.js

```diff
@@ -0,0 +1,78 @@
+import uuid from 'node-uuid';
+
```

Comment:
The registry here contains two properties: a `topMessage` (which is a single message) and `manageAndMonitorMessages` (which is an array of messages). I'm happy to reuse `ui/registry` if it can somehow accommodate both those properties.